products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp

```diff
@@ -76,7 +76,7 @@ namespace app::powernap
     void PowerNapProgressPresenter::onNapFinished()
     {
         if (alarmLightOnOffModel->getValue()) {
-            frontLightModel.restorePreviousState();
+            frontLightModel.revertUnsavedChanges();
```

Comment:
Worth testing if frontlight will still be turned on after power nap ends - i cannot do it on my Dev Kit P1 since the switches do not work